ui/i18n/src/main/res/values/strings.xml

```diff
@@ -841,4 +841,5 @@
     <string name="shortcut_subscription_label">Subscription shortcut</string>
     <string name="shortcut_select_subscription">Select subscription</string>
     <string name="add_shortcut">Add Shortcut</string>
+    <string name="hide_home_screen_warning">You are hiding your default page, another page will now be used as the default page</string>
```

Comment:
I'm not entirely sure if we need this. Simply using the first non-hidden page is probably the expected behavior, even without showing a dialog. I try to keep the number of popup dialogs as low as possible.

Reply:
@ByteHamster 
Addressed.